xml/gl.xml

```diff
@@ -9072,29 +9110,29 @@ typedef unsigned int GLhandleARB;
             <proto>void <name>glColorTableParameterfv</name></proto>
             <param group="ColorTableTarget"><ptype>GLenum</ptype> <name>target</name></param>
             <param group="ColorTableParameterPName"><ptype>GLenum</ptype> <name>pname</name></param>
-            <param group="CheckedFloat32" len="COMPSIZE(pname)">const <ptype>GLfloat</ptype> *<name>params</name></param>
+            <param kind="CheckedFloat32" len="COMPSIZE(pname)">const <ptype>GLfloat</ptype> *<name>params</name></param>
```

Comment:
This kind hasn't been declared at this top of the file.

Reply:
Some `CheckedFloat32` and `CheckedInt32` kinds will need to be turned into `Clamped[*]`, some may be more complex.
I proposed to leave them undocumented to say they are unsupported, while not deleting existing info from XML.
Similar in the case of `Path` and `FenceNV` kinds - but they need to turn into their respective `class` attribute.
Look at the "Long term (probably other PR's)" section of my review.